src/main/scala/com/cognite/sdk/scala/v1/Client.scala

```diff
@@ -154,7 +155,8 @@ class GenericClient[F[_]: Monad](
     val projectName: String,
     baseUrl: String =
       Option(System.getenv("COGNITE_BASE_URL")).getOrElse("https://api.cognitedata.com"),
-    auth: Auth = Auth.defaultAuth
+    auth: Auth = Auth.defaultAuth,
+    apiVersion: String = "v1"
```

Comment:
Hrm, on second thought, I'm not sure I like this. I think it would be better to have a com.cognite.sdk.scala.playground.Client (and/or GenericClient etc.) that inherits from v1 but adds experimental playground features like functions. Not a blocker for this PR, we can change it in a subsequent one.

Point is, someone importing from com.cognite.sdk.scala.v1 shouldn't risk using experimental features.

Reply:
That makes sense, I agree.